Cargo.toml

```diff
@@ -10,10 +10,10 @@ documentation = "https://docs.rs/syslog"
 keywords = ["syslog", "logs", "logging"]
 
 [dependencies]
-libc        = "^0.2"
-time        = "^0.1"
-log         = { version =  "^0.4", features = [ "std" ] }
-error-chain = { version = "^0.12", default-features = false }
+libc        = "0.2.67"
```

Comment:
I don't think anything else in this library uses `libc`, so I think it should be possible to just drop it?

Reply:
Unfortunately all os consts only live in `libc` not `std` and we leverage [`libc::EPROTOTYPE`](https://github.com/Geal/rust-syslog/blob/92a087169743699c489d6a5fd5c857017862b59a/src/lib.rs#L263).